docs/source/topic-guides/models.md

```diff
@@ -141,17 +141,17 @@ If there is only one _new owner_, the condition will be a simple signature condi
         },
         "uri": "<string>"
     },
-    "new_owners": ["<new owner public key>"]
+    "owners_after": ["<public key of the owner after the transaction happens>"]
```

Comment:
Oh, I was looking at this subsection more closely just now, and it's about conditions with just one "new owner" (the terminology of the section). In other words, it's about conditions, not transactions, and in the case of conditions, "new owner" is probably fine, so to keep this line consistent with the rest of the subsection, it should really be:
```json
    "owners_after": ["<new owner public key>"]
```